vagrant/provisioning/roles/confluent-platform-install/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# tasks file for confluent-install
+- name: Install curl and which
+  yum:
+    name: "{{ packages }}"
+  vars:
+    packages:
+    - curl
+    - which
+
+- name: Install Confluent Platform public key
+  rpm_key:
+    state: present
+    key: https://packages.confluent.io/rpm/5.4/archive.key
+
+- name: Create Confluent repo (1/2)
+  yum_repository:
+    name: Confluent.dist
+    description: Confluent repository (dist)
+    baseurl: https://packages.confluent.io/rpm/5.4/7
+    gpgcheck: yes
+    gpgkey: https://packages.confluent.io/rpm/5.4/archive.key
+    enabled: yes
+    file: confluent
+
+- name: Create Confluent repo (2/2)
+  yum_repository:
+    name: Confluent
+    description: Confluent repository
+    baseurl: https://packages.confluent.io/rpm/5.4
+    gpgcheck: yes
+    gpgkey: https://packages.confluent.io/rpm/5.4/archive.key
+    enabled: yes
+    file: confluent
+
+# The yum module does not support clearing yum cache in an idempotent way!
+- name: Clean the yum caches
+  command: yum clean all
+  args:
+    warn: False
+
+- name: Install Confluent Platform
+  yum:
+    name: confluent-platform-2.12
+    state: present
+  
```

Comment:
'2.12' is also ripe for extraction to the facts file.

We would like for Kafka to put its application (like log4j log files) logs into '{{root_folder}}/log/kafka', which according to https://serverfault.com/questions/831951/kafka-logfile-location you may be able to do by setting LOG_DIR environment variable in the SystemD unit file.  Please see if you can do this.

We would like Kafka to store its topic data in {{root_folder}}/arkcase/data/kafka, which according to https://docs.confluent.io/current/kafka/deployment.html may be done by setting logs.dir in config/server.properties.  Try and get this to work too.

Does installing confluent-platform-2.12 also enable the SystemD services and start the services?  If not you may need to add these steps.

Please check with Nadica Deleva on configuring the JMS adapter and the JDBC adapter.